integration_tests/e2e/list.cy.ts

```diff
@@ -1,13 +1,14 @@
 import { davidJones, fredMills, oscarJones } from '../../server/data/testData/offenderSearch'
 import ListPage from '../pages/nonAssociations/list'
-import { resetBasicStubs, navigateToDavidJonesNonAssociations } from './index'
 
 context('List non-associations page', () => {
   let listPage: ListPage
 
   beforeEach(() => {
-    resetBasicStubs()
-    listPage = navigateToDavidJonesNonAssociations()
+    cy.resetBasicStubs()
+    cy.navigateToDavidJonesNonAssociations().then(result => {
+      listPage = result
+    })
```

Comment:
Is there any advantage to this? I probably miss something but previous versions seemed simpler 😬

Reply:
i was reading cypress docs and the way the old version of `navigateToDavidJonesNonAssociations` was directly returning a value sounded incorrect because of how chaining works. we already had some "commands" and they're apparently the better way to group together other commands.

so now, the return type is a `Chainable<ListPage>` so you gotta use `.then(…)` to get at the value